src/nomad_simulations/schema_packages/general.py

```diff
@@ -265,16 +262,13 @@ def set_composition_formula(system_parent: ModelSystem) -> None:
     def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
         super(Schema, self).normalize(archive, logger)
 
-        # Finding which is the representative system of a calculation: typically, we will
-        # define it as the last system reported (TODO CHECK THIS!).
         # TODO extend adding the proper representative system extraction using `normalizer.py`
         if not self.model_system:
             logger.error('No system information reported.')
             return
-        system_ref = self.model_system[-1]
-        # * We define is_representative in the parser
-        # system_ref.is_representative = True
-        self.m_cache['system_ref'] = system_ref
+        # Set the representative system to last in list if not already set
```

Comment:
Pls add this specification to the pydocs too. We can later add it to the docs.
I suggest adding an `representative_system_index` to facilitate retrieval of the representative system.